frontend/src/assets/main.scss

```diff
@@ -16,6 +16,14 @@
   }
 }
 
+.content-center {
```

Comment:
you probably knew but primeflex (installed) has some built in css utility classes like `text-align` and `text-justify`. 
but i assume here we want to override into child components using !important.